tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -255,4 +253,32 @@ describe('data table', () => {
       order: SortingOrder.DESCENDING,
     });
   });
+
+  it('keeps sorting arrow invisible unless sorting on that header', () => {
+    const fixture = createComponent({
+      headers: [ColumnHeaders.VALUE, ColumnHeaders.RUN, ColumnHeaders.STEP],
+      sortingInfo: {header: ColumnHeaders.VALUE, order: SortingOrder.ASCENDING},
+    });
+    fixture.detectChanges();
+    const headerElements = fixture.debugElement.queryAll(By.css('th'));
+
+    expect(
+      window.getComputedStyle(
+        headerElements[1].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
+    ).toBe('1');
+    expect(
+      window.getComputedStyle(
+        headerElements[2].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
+    ).toBe('0');
+    expect(
+      window.getComputedStyle(
+        headerElements[3].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
+    ).toBe('0');
+  });
```

Comment:
Do we have tests for showing the icon when it is sorted on that header?

Another thing I'm wondering is, do we have tests for those icons? For example, when it's sorting on this header and it's ascending sorting, we should show down arrow_downward_24px (and we can expect getAttribute('svgIcon') to be that name)

Reply:
I am not sure what all you are asking.

I added a check to ensure the upward arrow is there.  I also added a test for the downward arrow when order is DESCENDING. 

Does that satisfy this comment?